keras_core/ops/numpy.py

```diff
@@ -146,8 +146,20 @@
 
 
 def broadcast_shapes(shape1, shape2):
-    # Broadcast input shapes to a unified shape.
-    # Convert to list for mutability.
+    """Broadcast input shapes to a unified shape.
+    Convert to list for mutability.
```

Comment:
Add line break above

Reply:
done, I've added blank line above "convert to list ..."